tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -69,11 +69,11 @@ def test_ebs_snapshot(
 
 
 # cn-north-1 does not support KMS
-@pytest.mark.regions(["ca-central-1", "us-gov-east-1"])
+@pytest.mark.regions(["us-gov-east-1"])
 @pytest.mark.instances(["c5.xlarge"])
-@pytest.mark.schedulers(["slurm", "awsbatch"])
-@pytest.mark.usefixtures("region", "os", "instance")
-@pytest.mark.skip_oss(["centos6"])
+@pytest.mark.schedulers(["slurm"])
```

Comment:
I'd rather leave awsbatch here and remove it from the ebs_single but then we can't run this in gov-cloud

Reply:
ok, done